pkg/config/squash.go

```diff
@@ -130,7 +130,7 @@ func (s *Squash) GetDebugTargetContainerFromSpec(dbt *DebugTarget) error {
 	for _, podContainer := range dbt.Pod.Spec.Containers {
 		log.Debug(podContainer.Image)
 		log.Info(podContainer.Image)
-		if strings.HasPrefix(podContainer.Image, s.Container) {
+		if strings.HasPrefix(podContainer.Name, s.Container) {
```

Comment:
would you mind also updating line 139 to 
```go
return errors.New(fmt.Sprintf("such container found with name %v", s.Container))
```
for clarity

Reply:
Done.